Resources/Views/Submissions/Fields/password-input.leaf

```diff
@@ -0,0 +1,19 @@
+<div class="form-group">
```

Comment:
Do we leave these duplicated leaf files for now? Or create one and point all of the config to that file?

Reply:
Created one for now and then let's see if we need to split in the future.